LICENSE.md

```diff
@@ -1,31 +1,21 @@
-3-clause BSD license
-====================
 
-Copyright 2012-2014 Motorola Mobility LLC, Montage Studio Inc, and contributors.
-All rights reserved.
+Copyright 2012 Kristopher Michael Kowal. All rights reserved.
```

Comment:
Indeed, this is not appropriate. Motorola and Montage Studio paid for this and own the copyright. It would be inappropriate even to claim that I was the primary author since I built this around the array change kernel from the previous iteration of Montage’s binding system as had been touched by most of the team before me.